SerialPrograms/Source/CommonFramework/Options/EditableTableOption.cpp

```diff
@@ -90,6 +90,34 @@ template<typename T> QWidget* make_integer_table_cell(QWidget& parent, T& value)
     return box;
 }
 
+
+QWidget* make_limited_integer_table_cell(QWidget& parent, uint8_t& value, uint8_t min, uint8_t max){

```

Comment:
Do we need to limit this to only 8-bit integers?

Reply:
changed to int :)